app/components/SUSI/SIgnUpButton.tsx

```diff
@@ -0,0 +1,8 @@
+import { Button} from '@mui/material'
+
+
+export const SignUpButton = () => {
+	return (
+			<Button sx={{padding: '12px 52px', backgroundColor: '#0265DC', color: '#FFF', marginTop: '26px', marginBottom: '16px'}}>SIGN UP</Button>
+	)
+}
```

Comment:
do you have eslint or prettier installed? it should autoformat this

Reply:
installed